<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>show2D</class>
 <widget class="QWidget" name="show2D">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>left:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>backï¼š</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTextBrowser" name="textBrowser_back"/>
     </item>
     <item row="3" column="0">
      <widget class="QTextBrowser" name="textBrowser_left"/>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>up:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QTextBrowser" name="textBrowser_front"/>
     </item>
     <item row="3" column="2">
      <widget class="QTextBrowser" name="textBrowser_right"/>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>front:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>right:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QTextBrowser" name="textBrowser_up"/>
     </item>
     <item row="7" column="1">
      <widget class="QTextBrowser" name="textBrowser_down"/>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>down:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
